test/system/211-dbus.bats

```diff
@@ -87,3 +87,311 @@ teardown() {
   # shellcheck disable=SC2154
   assert [ ${#stderr_lines[@]} -eq 0 ]
 }
+
+@test "dbus: session bus inside Arch Linux" {
+  create_distro_container arch latest arch-toolbox-latest
+  expected_response="$(gdbus call \
```

Comment:
Don't you want to mark `expected_response` as a local variable with:
```
local expected_response
```

... like you did before?